src/style.css

```diff
@@ -1 +1,98 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 400;
+}
+
+html {
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.searchBar {
+  display: flex;
+  position: absolute;
+  align-items: center;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 100%;
+  margin-left: 48px;
+}
+
+.searchBar--big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 17px;
+}
```

Comment:
It looks like you have duplicate CSS rules in your `.searchBar--big` and `.searchBar` selectors. You're setting the same `margin-top`, `margin-bottom`, and `margin-left` properties in both selectors. Consider removing the duplicate rules from the `.searchBar--big` selector to keep your CSS DRY (Don't Repeat Yourself).